static/css/contact-us.less

```diff
@@ -3,10 +3,20 @@
    Contact us page.
    ========================================================================== */
 
-.helpful-terms_header {
-    .h5();
-    margin-right: unit(6px / @font-size, em);
-    vertical-align: middle;
+.helpful-terms {
+    &_header {
+        .h5();
+        margin-right: unit(6px / @font-size, em);
+        vertical-align: middle;
+    }
+
+    &_list {
```

Comment:
I'm reluctant to include mixins where they could be classes on the elem since it increases the duplication of code. Not sure what our distinctions of classes vs mixins are, but this is an area of Less syntax I really dislike since all classes are technically mixins.